### PR TITLE
Update the name of pallet contracts storage item

### PR DESCRIPTION
**Pull request by h4nsu**
They didn't care to update that call...

### CODE REVIEW

File: packages/page-contracts/src/Contracts/Summary.tsx [hunk 1]
```diff
@@ -18,7 +18,7 @@ interface Props {
 function Summary ({ trigger }: Props): React.ReactElement<Props> {
   const { t } = useTranslation();
   const { api } = useApi();
-  const accountCounter = useCall<BN>(api.query.contracts.accountCounter);
+  const accountCounter = useCall<BN>(api.query.contracts.nonce);
```
Comment (pmikolajczyk41) [inline packages/page-contracts/src/Contracts/Summary.tsx:21]: optionally change the var name

**Review by pmikolajczyk41 — APPROVED** [review]

File: packages/page-contracts/src/Contracts/Summary.tsx (hunk 1 above)
Comment (piotrMocz) [reply packages/page-contracts/src/Contracts/Summary.tsx:21]: +1

**Review by piotrMocz — APPROVED** [review]
Beautiful 😊

File: packages/page-contracts/src/Contracts/Summary.tsx (hunk 1 above)
Comment (h4nsu) [reply packages/page-contracts/src/Contracts/Summary.tsx:21]: The old name is more descriptive ^^

**Pull request merged**